frontend/taipy-gui/src/components/Taipy/TableFilter.tsx

```diff
@@ -92,7 +94,14 @@ const getActionsByType = (colType?: string) =>
     (colType && colType in actionsByType && actionsByType[colType]) ||
     (colType === "any" ? { ...actionsByType.string, ...actionsByType.number } : actionsByType.string);
 
-const getFilterDesc = (columns: Record<string, ColumnDesc>, colId?: string, act?: string, val?: string) => {
+const getFilterDesc = (
+    columns: Record<string, ColumnDesc>,
+    colId?: string,
+    act?: string,
+    val?: string,
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
remove this 
If we don't use it, it *is* an error ?

Reply:
matchCase should be part of the returned object

Reply:
> remove this If we don't use it, it _is_ an error ?

ESlint throws an error of unused variable.

Reply:
well it means that you should use it !